api/external/data_feed/data_feed.proto

```diff
@@ -66,20 +69,58 @@ service DatafeedService {
 			action: "datafeed:destinations:test"
 		};
 	}
+	rpc EnableDestination(UpdateDestinationEnableRequest) returns (GetDestinationResponse) {
+		option (google.api.http) = {
+			patch: "/api/v0/datafeed/destination/enable/{id}"
+			body: "*"
+		};
+		option (chef.automate.api.iam.policy) = {
+			resource: "datafeed:destinations:{id}"
+			action: "datafeed:destinations:update"
+		};
+	}
+	rpc DestinationConfig(ConfigRequest) returns (ConfigResponse) {
```

Comment:
Why is this `DesticationConfig` ? According to me it should be `GlobalDataFeedConfig`.
And respectively these can be used:
`GlobalDataFeedConfigRequest`
`GlobalDataFeedConfigResponse`